docs/source/examples/external_sp2_output.txt

```diff
@@ -1,10 +1,10 @@
 
 Cost
 ----
- 0.7854
+ 0.7071
 
 Free Variables
 --------------
 x : 0.7854
-y : 0.7854
+y : 0.7071
```

Comment:
Yeah, it's a bit non-obvious. I was annoyed at this 'regression' until I stared at it enough